invenio_saml/ext.py

```diff
@@ -1,21 +1,239 @@
 # -*- coding: utf-8 -*-
 #
-# Copyright (C) 2019 Esteban J. Garcia Gabancho.
+# Copyright (C) 2019, 2022 Esteban J. Garcia Gabancho.
 #
 # Invenio-SAML is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
 """Invenio module that provides SAML integration."""
 
-from __future__ import absolute_import, print_function
+import json
+from collections.abc import Mapping
+from functools import wraps
 
-from flask_sso_saml import FlaskSSOSAML
+from flask import url_for
+from onelogin.saml2.idp_metadata_parser import OneLogin_Saml2_IdPMetadataParser
+from werkzeug.utils import cached_property, import_string
 
-from . import config
+from invenio_saml import config
```

Comment:
Minor: is there a reason not to use the relative import for package specific imports?

Reply:
That's just my editor taking over control, I'll revert the change ;-)